Main/ClientApp/src/Infrastructure/UrlUtilities.ts

```diff
@@ -21,3 +21,23 @@ export function OpenInNewTab(href: string) {
     rel: "noopener noreferrer",
   }).click();
 }
+
+export function OpenLink(href: string) {
+  Object.assign(document.createElement("a"), {
+    href,
+  }).click();
+}
+
+// Get the hash of the url
```

Comment:
I am having trouble understanding what this does and why this is needed. Can you provide a better comment? 
Also for method name: GetUrlHash

Reply:
this function is copied from an example and maps query params from url given after hash symbol (#)